.github/workflows/virusTotalSubmit.js

```diff
@@ -0,0 +1,105 @@
+const glob = require('glob');
+const { v4: uuidv4 } = require('uuid');
+const fs = require('fs');
+const { exec } = require('child_process');
+
+
+function sleep(n) {
+  Atomics.wait(new Int32Array(new SharedArrayBuffer(4)), 0, 0, n);
+}
+
+
+function countAPIUsageAndWait({core}) {
+  // Sleep 20 seconds to avoid rate limiting
+  sleep(20 * 1000);
+  core._apiUsageCount++;
+}
+
+
+function submitAddon({core}, addonMetadata, downloadFileName) {
```

Comment:
Could you please document the expected outcomes here? Is the postcondition that `vt` will set an exit code that we can use to determine the scan result?

Reply:
Not quite, we fetch scan results later on. I've added more comments to explain this in more detail.